Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:14 as base
```

Comment:
I think this is okay for now. But when we start deploying this, I think we should consider using node:14-alpine. It's a much smaller version of the node image which would be good. However, because it's a smaller image, it does take away some libraries. You can read more about it here https://hub.docker.com/_/node but I do not think we necessarily need it. But node:14 is fine for now.

Reply:
Might as well switch it now